src/frontend/pages/recruitment/DecisionPage/DecisionPage.js

```diff
@@ -160,10 +162,10 @@ const DecisionPage = () => {
   };
 
   // TODO: Replace the mock data below with actual data taken from the appropriate sources.
-  const mockData = {
+  const userData = {
     // Extracted from Google OAuth.
-    execName: 'John Doe',
-    execEmail: 'john.doe@waterloop.ca',
+    execName: profileData?.name || '',
+    execEmail: profileData?.email || '',
     execPhoneNum: '(000) 000-0000',
     // Extracted from the decision portal table row.
     position: 'Fullstack Developer',
```

Comment:
What are we going to do about the reamining hard-coded data ?

Reply:
This will be addressed in future PRs as the other hardcoded fields depend on either the configuration page info (meeting links + deadlines) or application data itself (Subteam + position)